docs/draft/webview.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Dapp browser API usage
+version: 0.1.0
+status: Draft
+authors: 
+---
+
+# Dapp browser API usage
+    
+##  Table of Contents
+
+1. [Abstract](#abstract)
+2. [Definitions](#definitions)
+3. [Implementation](#implementation)
+4. [Usage](#usage)
+    * [Properties](#properties)
+        * [`isStatus`](#isStatus)
+        * [`status`](#status)
+    * [Methods](#methods)
+        * [`enable`](#enable)
+        * [`isConnected`](#isConnected)
+        * [`scanQRCode`](#scanQRCode)
+        * [`send`](#send)
+        * [`sendAsync`](#sendAsync)
+        * [`sendSync`](#sendSync)
+5. [Compatibility](#compatibility)
+6. [Changelog](#changelog)
+7. [Acknowledgements](#acknowledgements)
+8. [Copyright](#copyright)
+9. [Footnotes](#footnotes)
+
+## Abstract
+
+Status exposes an Ethereum Provider object (`window.ethereum`) to JS code running inside the browser. It also provides bidirectional communication between JS code and Status.
+
+## Definitions
+
+| Term       | Description                                                                         |
+|------------|-------------------------------------------------------------------------------------|
+| **Webview**   | React Native wrapper for a platform-specific browser core implementation.                                        |
+| **Ethereum Provider** | A JS object (`window.ethereum`) injected into each web page opened in the browser providing web3 compatible provider. |
+| **Bridge** | A set of facilities allow bidirectional communication between JS code and Status. |
+
+## Implementation
```

Comment:
Would be good if we could lead with INTERFACE requirements and then talk about a specific implementation as an example, as opposed to the only way to implement the Status protocol.

Reply:
Agree, made some changes, tried to be more generic.